packages/styles/scss/patterns/sub-patterns/content-group/_content-group.scss

```diff
@@ -22,31 +20,40 @@
         margin-bottom: $carbon--layout-05;
       }
     }
-
     @include carbon--breakpoint(lg) {
       padding-top: $carbon--layout-05;
       padding-bottom: $carbon--layout-07;
-
       &__title {
         margin-bottom: $carbon--layout-03;
       }
     }
 
     &__title {
-      padding-left: $carbon--layout-01;
       padding-right: $carbon--layout-01;
       max-width: 90%;
 
       @include carbon--type-style('expressive-heading-04', true);
     }
 
     &__children {
-      padding-left: 0;
-      padding-right: 0;
+      margin-bottom: $carbon--layout-03;
+    }
+
+    &__CTA {
```

Comment:
Also, the card is going well beyond the 320px max-width. (this maybe something you can apply to the card. I thought it was already applied for something)

<img src="https://user-images.githubusercontent.com/3793636/74361394-744a2d80-4d8c-11ea-926e-5ee84525c0cb.png" alt="" width="300px" />

Reply:
do I need to add max-width for CTA?